### PR TITLE
bitbucket-server-post-build-status 0.0.1

### PR DESCRIPTION
**Pull request by teameh**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by teameh** [comment]
Not sure if I need a test for this step.

File: steps/bitbucket-server-post-build-status/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: false
+inputs:
+- bitbucket_server_domain: $BITBUCKET_SERVER_DOMAIN
```
Comment (koral--) [inline steps/bitbucket-server-post-build-status/0.0.1/step.yml]: `BITBUCKET_SERVER_DOMAIN` variable is not exposed by Bitrise CI nor by any step in steplib, so it should not be used.

File: steps/bitbucket-server-post-build-status/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  opts:
+    is_required: true
+    summary: Full domain name without protocol eg 'my-domain.com'
+    title: Bitbucket Server domain name
+- bitbucket_server_username: $BITBUCKET_SERVER_USERNAME
+  opts:
+    is_required: true
+    summary: Username used to make REST calls
+    title: Bitbucket Server username
+- bitbucket_server_password: $BITBUCKET_SERVER_PASSWORD
```
Comment (koral--) [inline steps/bitbucket-server-post-build-status/0.0.1/step.yml]: variable is not exposed by Bitrise CI nor by any step in steplib

File: steps/bitbucket-server-post-build-status/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: false
+inputs:
+- bitbucket_server_domain: $BITBUCKET_SERVER_DOMAIN
+  opts:
+    is_required: true
+    summary: Full domain name without protocol eg 'my-domain.com'
+    title: Bitbucket Server domain name
+- bitbucket_server_username: $BITBUCKET_SERVER_USERNAME
```
Comment (koral--) [inline steps/bitbucket-server-post-build-status/0.0.1/step.yml]: variable is not exposed by Bitrise CI nor by any step in steplib

File: steps/bitbucket-server-post-build-status/0.0.1/step.yml (hunk 1 above)
Comment (teameh) [reply steps/bitbucket-server-post-build-status/0.0.1/step.yml]: Check. Changed in 15a5aa3

File: steps/bitbucket-server-post-build-status/0.0.1/step.yml (hunk 2 above)
Comment (teameh) [reply steps/bitbucket-server-post-build-status/0.0.1/step.yml]: Fixed

File: steps/bitbucket-server-post-build-status/0.0.1/step.yml (hunk 3 above)
Comment (teameh) [reply steps/bitbucket-server-post-build-status/0.0.1/step.yml]: Fixed

**Comment by teameh** [comment]
Closed in favor of 
https://github.com/bitrise-io/bitrise-steplib/pull/2319

**Pull request closed without merging**